### DISCUSSION TITLE
Sort imports in entire notebook

### DISCUSSION THREAD

**Discussion by benlindsay** [microsoft/pylance-release#2633] [Enhancement]
Running `python.sortImports` in a single cell of a notebook works great, and I'd love to be able to run that command or a similar command on the entire notebook. I love the formatting button provided by `jupyterlab_code_formatter` which runs black and isort on every cell, and am hoping to be able to do the same within VSCode. I [brought this up](https://github.com/microsoft/vscode-jupyter/issues/5705#issuecomment-1057335505) in the vscode-jupyter repo, but they suggested I should open an issue here since functionality decisions might be complicated. I would be opposed to pulling all imports to the top cell, and would instead prefer to sort imports in each cell independently, much like how `jupyterlab_code_formatter` works. Is this something that would make sense to implement within pylance?

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
